src/assets/i18n/en.json

```diff
@@ -18,6 +18,34 @@
     "ANALYTICS_FALLBACK": "Some analytics here in the future",
     "SIDENAV": {
       "SLOTS": "SLOTS"
+    },
+    "CREATE_TIME_SLOT_CARD": {
+      "START_TIME": "Start Time",
+      "DURATION": "Duration",
+      "ERROR": "Slot will end in the next day"
+    },
+    "TIME_SLOT":{
+      "LESSON": "Piano",
```

Comment:
LESSON_TYPE

Reply:
Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38